hecuba_py/src/hset.py

```diff
@@ -0,0 +1,421 @@
+from IStorage import IStorage, AlreadyPersistentError
+from hecuba import config, log
+from collections import namedtuple
+import re
+import uuid
+
+class Iterator:
```

Comment:
remove this class. Use a the function map instead to select what you need.
e.g.
```python
it = iter_of_something()
map(lambda a:a[0],it)
```